testui/elements/testui_element.py

```diff
@@ -2,11 +2,14 @@
 import os
 
 from os import path
+from typing import List
 
 from appium.webdriver.common.touch_action import TouchAction
+from appium.webdriver.webelement import WebElement
 from selenium.webdriver.common.by import By
 
 from testui.support import logger
+from testui.support.testui_driver import TestUIDriver
```

Comment:
Dont import this, it will generate a looped import and you wont be able to run it when using it as dependency